src/pages/identity/IdentityDetails.css

```diff
@@ -0,0 +1,91 @@
+@import "@nomios/web-uikit/styles/colors";
+@import "@nomios/web-uikit/styles/mixins/typography";
+
+.detailsWrapper {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+
+    & .avatar {
+        margin-bottom: 1rem;
+        font-size: 2.5rem;
+    }
+
+    & .name {
+        z-index: 2;
+        margin: 0;
+        margin-bottom: 1.5rem;
+        color: var(--color-white);
+    }
+
+    & .did {
+        @mixin typography-body-2 rem;
+        margin-bottom: 6rem;
+        opacity: 0.6;
+        color: var(--color-white);
+    }
+
+    & .lowerBar {
+        width: 100%;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+
+        & .attributes {
+            width: 76%;
+            min-width: 58rem;
+            max-width: 78rem;
+            display: flex;
+            flex-direction: row;
+
+            & .attribute {
+                opacity: 0.8;
+
+                &:first-of-type {
+                    flex-shrink: 0;
+                }
+
+                &:not(:last-of-type) {
+                    margin-right: 7%;
+                }
+
+                & .title {
+                    @mixin typography-captions rem;
+                    margin-bottom: 1rem;
+                    color: var(--color-tan);
+                }
+
+                & .value {
+                    color: var(--color-white);
+                }
+
+            }
+        }
+
+        & .editButton {
+            min-width: 16.5rem;
+            margin-left: 5rem;
+
+            & .buttonIcon {
+                font-size: 1.6rem;
+            }
+        }
+    }
+}
+
+@media (max-width: 1280px) {
```

Comment:
Shouldn't we be using `custom-medias` here? Maybe we should add them to [nomios-web-uikit](https://github.com/ipfs-shipyard/nomios-web-uikit) and import them here.

//cc @satazor

Reply:
I was honestly unfamiliar with `custom-medias`, so I don't really have an opinion here. Wouldn't mind to hear another piece of feedback, in case of consensus I can change it.

Reply:
If we are going to use this media query often, then we should create a custom media. Please ask Sousa if this is something just for this page or not.